pkgs/development/python-modules/treeo/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, poetry
+, jax
+, jaxlib
+}:
+
+buildPythonPackage rec {
+  pname = "treeo";
+  version = "0.0.9";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "cgarciae";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "0jfqj5150braj4ybnifc6b8mp0w2j93li6bm20lcd7a19qs0lkk2";
+  };
+
+  buildInputs = [
+    poetry
+  ];
```

```suggestion
  nativeBuildInputs = [
    poetry-core
  ];
```
https://github.com/cgarciae/treeo/blob/a402f3f69557840cfbee4d7804964b8e2c47e3f7/pyproject.toml#L30

---

Done.